site/en/r1/guide/checkpoints.md

```diff
@@ -63,9 +63,6 @@ classifier.train(
 As suggested by the following diagrams, the first call to `train`
 adds checkpoints and other files to the `model_dir` directory:
 
-<div style="width:80%; margin:auto; margin-bottom:10px; margin-top:20px;">
-<img style="width:100%" src="../images/first_train_calls.png">
```

Comment:
Let's not delete the images here. This can be changed to https://www.tensorflow.org/images/first_train_calls.png

Reply:
Reverse the image tag and apply it.
Thanks